share/spice/piratetalk/piratetalk.js

```diff
@@ -0,0 +1,28 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_piratetalk = function(api_result) {
+
+        if (!api_result || api_result.translation==="") {
+            return Spice.failed('piratetalk');
+        }
+        
+        Spice.add({
+            id: "piratetalk",
+            name: "Answer",
+            data: {
+                title:api_result.translation,
+            },
+            meta: {
+                sourceName: "Pirate",
+                sourceUrl: 'https://pirate-speak.herokuapp.com/',
+            },
+            templates: {     
+                group: 'base',
```

Comment:
For now let's keep things simple and stick to the text template. We only use the base template when needed and in this case the design team has a special design for the quote IA.

Any new IA PRs should use a built-in template unless our design team wants something custom